src/actions/receive-profile.js

```diff
@@ -1020,7 +1020,9 @@ export function getProfilesFromRawUrl(
         await dispatch(retrieveProfileFromStore(pathParts[1], true));
         break;
       case 'from-url':
-        await dispatch(retrieveProfileOrZipFromUrl(pathParts[1], true));
+        await dispatch(
+          retrieveProfileOrZipFromUrl(decodeURIComponent(pathParts[1]), true)
```

Comment:
This mimics what we have in https://github.com/firefox-devtools/profiler/blob/master/src/app-logic/url-handling.js#L355.

I wonder if we shouldn't try to reuse some code more...